src/components/bottom_bar/bottom_bar.styles.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css, keyframes } from '@emotion/react';
+import { UseEuiTheme } from '../../services';
+import { euiShadowFlat } from '../../themes/amsterdam/global_styling/mixins';
+
+//const _euiBottomBarColor = '#131317';
+const _euiBottomBarZHeader = 1000;
+
+const euiBottomBarAppear = keyframes`
+0% {
+    transform: translateY(100%);
+    opacity: 0;
+  }
+
+  100% {
+    transform: translateY(00%);
+    opacity: 1;
+  }
+`;
+
+export const euiBottomBarStyles = ({ euiTheme, colorMode }: UseEuiTheme) => ({
+  // Base
+  euiBottomBar: css`
+    ${euiShadowFlat(euiTheme, undefined, colorMode)};
+    background: ${euiTheme.colors.lightShade};
```

Comment:
We still need this background color to render as it did previously. The sass variable is: 
```scss
$euiHeaderDarkBackgroundColor: lightOrDarkTheme(shade($euiColorDarkestShade, 28%), shade($euiColorLightestShade, 50%)) !default;
```

But since we're only ever in `dark` theme, we just need to apply the second parameter as the color:

```suggestion
    background: ${shade(euiTheme.colors.lightestShade, .5)};
```